bin/foreman_scap_client_bash

```diff
@@ -0,0 +1,262 @@
+#!/usr/bin/env bash
+
```

Comment:
Could we drop `bash` from the name of the executable?

Reply:
Sure, I named it more explicitly since there is no migration path yet, so we don't conflict with the current script.